compose/cli/main.py

```diff
@@ -238,15 +238,15 @@ def rm(self, project, options):
         Usage: rm [options] [SERVICE...]
 
         Options:
-            --force   Don't ask to confirm removal
-            -v        Remove volumes associated with containers
+            -f, --force   Don't ask to confirm removal
+            -v            Remove volumes associated with containers
         """
         all_containers = project.containers(service_names=options['SERVICE'], stopped=True)
         stopped_containers = [c for c in all_containers if not c.is_running]
 
         if len(stopped_containers) > 0:
             print("Going to remove", list_containers(stopped_containers))
-            if options.get('--force') \
+            if options.get('--force') or options.get('-f') \
```

Comment:
No need. You can make a change while keeping it to one commit with `git commit --amend`, and then push that to your branch with `git push --force`.